CMakeLists.txt

```diff
@@ -61,7 +68,8 @@ include_directories(${STRING_THEORY_INCLUDE_DIRS})
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
 include_directories(${GLEW_INCLUDE_DIR})
 include_directories(${OPENGL_INCLUDE_DIR})
-include_directories(${SDL2_INCLUDE_DIRS})
+include_directories(${SDL2_INCLUDE_DIR})
```

Comment:
Is this still needed when using the import targets (`target_link_libraries( ... SDL2::SDL2)`)?  Often those include setting the lib's include path...

Reply:
Yup!  Good call.  I just had to add it individually to the projects which use it.